pepper/cli.py

```diff
@@ -140,6 +140,14 @@ def add_tgtopts(self):
                 action='store_const', const='grain_pcre',
             help="Target based on PCRE matches on system properties")
 
+        optgroup.add_option('-I', '--pillar', dest='expr_form',
+                action='store_const', const='pillar',
+            help="Target based on pillar values")
+
+        optgroup.add_option('--pillar-pcre', dest='expr_form',
```

Comment:
I didn't add a short option `-J` here by analogy with `--grain-pcre` not having `-P`.  I don't see why they both couldn't have them, though.